database/migrations/2023_05_14_132500_update_event_registration_add_position_id_detail_column.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        Schema::table('event_registration', function ($table) {
+            $table->string('position_id_detail', 100)->default('');
```

Comment:
Hmm... I'm not sure if I can do that or not. Since the 'position_id_detail' is being displayed in a input text field I'm not sure what happens if it is null. I thought an empty string would be a better initialization value. Let me play around with it and I'll find out.

Reply:
Ok, that appears to work just fine. Change made and I QA'd it (currently loaded on QA-2)